server/Makefile

```diff
@@ -0,0 +1,56 @@
+include build.properties
+export $(shell sed 's/=.*//' build.properties)
+
+IMAGE_NAME							:=alfresco/alfresco-identity-service:$(IDENTITY_VERSION)
+IMAGE_NAME_WITH_BASE_OS				:=$(IMAGE_NAME)-java-11-openjdk-centos-7
+IMAGE_NAME_WITH_BASE_OS_AND_SHA		:=$(IMAGE_NAME_WITH_BASE_OS)-$(shell git rev-parse HEAD | cut -c1-12)
+export IMAGE_NAME_WITH_BASE_OS_AND_SHA
+
+THEME_VERSION	:=0.1
+export THEME_VERSION
+
+# Alfresco keycloak variables
+KEYCLOAK_DIST	:=https://github.com/Alfresco/keycloak/releases/download/$(KEYCLOAK_VERSION)/keycloak-$(KEYCLOAK_VERSION).zip
+# Dev properties
+GIT_REPO		:=Alfresco/keycloak
+GIT_BRANCH		:=7.0.1/AUTH-328-library-patch
```

Comment:
This will need to be changed before merging, right?

Reply:
These two properties are not used by default. They are there for development time.
The branch should be specified by the developer, it is there for an example.

Reply:
Done